### ISSUE TITLE
pactbroker.host and other depcrated properties still in use

### ISSUE THREAD

**Issue by gedankennebel** [pact-foundation/pact-jvm#1487]
With this change the pactbroker.host got deprecated and pactbroker.url should be set instead:
https://github.com/pact-foundation/pact-jvm/blob/ac6a0eae0b18183f6f453eafddb89b90741ace42/provider/src/main/java/au/com/dius/pact/provider/junitsupport/loader/PactBroker.java#L33

However, some readme and test properties files still use host and it is not clear from the docs and examples that we should rather use url. This is kinda confusing. 
See for example: 
https://github.com/pact-foundation/pact-jvm/blob/ac6a0eae0b18183f6f453eafddb89b90741ace42/provider/junit5spring/src/test/resources/application.yml
https://github.com/pact-foundation/pact-jvm/blob/ac6a0eae0b18183f6f453eafddb89b90741ace42/provider/junit5spring/README.md

**Comment by ChrisJBurns** [comment]
Yes, this is also confusing for me since I am using `url` which is the recommended field to use, yet I get errors in my logs saying that I'm not using `host`:
```
java.lang.IllegalArgumentException: Invalid pact broker host specified ('${pactbroker.host:}'). Please provide a valid host or specify the system property 'pactbroker.host'.
```

**Comment by krizajb** [comment]
Facing with the same issue today. url is being set, however host is trying to be used.
